libclc/generic/lib/clcmacro.h

```diff
@@ -140,6 +140,7 @@
     return (RET_TYPE##16)(FUNCTION(x, y, z.lo), FUNCTION(x, y, z.hi));         \
   }
 
+
```

Comment:
This blank line looks suspicious - I don't see a double blank line locally or upstream. Is this an artifact of a conflict resolution?